qa-tests-backend/src/main/groovy/objects/Rsyslog.groovy

```diff
@@ -0,0 +1,56 @@
+package util
```

Comment:
This file is in the wrong location for this package.

Move it to qa-tests-backend/src/main/groovy/util